ui/css/design-system/_colors.scss

```diff
@@ -16,12 +16,10 @@ $color-map: (
   'primary-alternative': --color-primary-alternative,
   'primary-muted': --color-primary-muted,
   'primary-inverse': --color-primary-inverse,
-  'primary-disabled': --color-primary-disabled,
   'error-default': --color-error-default,
   'error-alternative': --color-error-alternative,
   'error-muted': --color-error-muted,
   'error-inverse': --color-error-inverse,
-  'error-disabled': --color-error-disabled,
```

Comment:
No instances of error-disabled exist in code base

![Screenshot 2024-05-31 at 4 29 08 PM](https://github.com/MetaMask/metamask-extension/assets/8112138/e1b66308-f685-48e8-b440-5b3f2101dffe)